Code2Gether-Discord-Bot.Library/BusinessLogic/PingLogic.cs

```diff
@@ -19,7 +20,7 @@ public PingLogic(ILogger logger, ICommandContext context, int latency)
             _latency = latency;
         }
 
-        public Embed Execute()
+        public Task<Embed> Execute()
```

Comment:
This should be `public async Task<Embed> ExecuteAsync` since there are awaiters here.